Azure/sonic-mgmt
# [mutli-asic][vs]: Add support to test multi-asic vs with four asic and 8 interfaces

### Description of PR


Summary:
Fixes # (issue)

### Type of change




### Approach
#### What is the motivation for this PR?
- https://github.com/Azure/sonic-buildimage/pull/6558 will add new hwsku msft_four_asic_vs which is a 4 asic hwsku with 8 front panel interfaces. With this change, add supporting files to sonic-mgmt so that the new 4 asic hwsku testbed can be brought up.

#### How did you do it?
- Add new t1-lag topo file to support 8 front panel interfaces.
-  Modify sonic.xml.j2 file to use higher number of vcpus based on the multi-asic hwsku, 6 asic hwsku will require more vcpus.
- Add new testbed for msft_four_asic_vs and new DUT vlab-08.
- Add a new minigraph for mutli-asic vlab-08   DUT to come up with t1-lag topology.
- Start topology service before load_minigraph for multi-asic vs platform.

#### How did you verify/test it?
- Bring up multi-asic  vs image with 4-asics testbed using add-topo:
 ./testbed-cli.sh -t vtestbed.csv -m veos_vtb -k ceos add-topo vms-kvm-four-asic-t1-lag password.txt
 - Copy the minigraph to virtual switch.
 - Start topology service (systemctl start topology.service)
 - load minigraph.
 - Ensure that all internal and external interfaces are up and all BGP sessions are up.
```
 admin@vlab-08:~$ show ip bgp summary -d all

IPv4 Unicast Summary:
asic0: BGP router identifier 8.0.0.0, local AS number 65100 vrf-id 0
BGP table version 12754
asic1: BGP router identifier 8.0.0.1, local AS number 65100 vrf-id 0
BGP table version 12775
asic2: BGP router identifier 8.0.0.4, local AS number 65100 vrf-id 0
BGP table version 6398
asic3: BGP router identifier 8.0.0.5, local AS number 65100 vrf-id 0
BGP table version 6398
RIB entries 51176, using 9416384 bytes of memory
Peers 14, using 292880 KiB of memory
Peer groups 12, using 768 bytes of memory


Neighbhor      V     AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ  Up/Down      State/PfxRcd  NeighborName
-----------  ---  -----  ---------  ---------  --------  -----  ------  ---------  --------------  --------------
10.0.0.1       4  65200       3774       6963         0      0       0  00:29:50             6370  ARISTA01T2
10.0.0.5       4  65200       3775       6914         0      0       0  00:29:46             6370  ARISTA03T2
10.0.0.33      4  64001        603       6980         0      0       0  00:29:54                3  ARISTA01T0
10.0.0.35      4  64002        603       6978         0      0       0  00:29:52                3  ARISTA02T0
10.0.0.37      4  64003        604       6976         0      0       0  00:29:51                4  ARISTA03T0
10.0.0.39      4  64004        602       6978         0      0       0  00:29:48                3  ARISTA04T0
10.1.0.0       4  65100       3769       3765         0      0       0  00:28:32             6398  ASIC2
10.1.0.1       4  65100       3766       3770         0      0       0  00:28:36             6377  ASIC0
10.1.0.2       4  65100       3768       3763         0      0       0  00:28:27             6398  ASIC3
10.1.0.3       4  65100       3765       3770         0      0       0  00:28:34             6377  ASIC0
10.1.0.4       4  65100       3770        586         0      0       0  00:28:34             6398  ASIC2
10.1.0.5       4  65100        587       3770         0      0       0  00:28:36               21  ASIC1
10.1.0.6       4  65100       3770        586         0      0       0  00:28:35             6398  ASIC3
10.1.0.7       4  65100        588       3772         0      0       0  00:28:40               21  ASIC1
admin@vlab-08:~$ show platform summary
Platform: x86_64-kvm_x86_64-r0
HwSKU: msft_four_asic_vs
ASIC: vs
ASIC Count: 4
```
#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
